README.md

```diff
@@ -194,6 +290,18 @@ Curated database of foundation models for robotics
 
 ---
 
+### **ELLMER**
```

Comment:
Paper link is broken. add more comments on it. what is the added value? how it incorporates force?